targets/TARGET_ARM_SSG/TARGET_MUSCA_S1/serial_api.c

```diff
@@ -0,0 +1,215 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2017-2019 Arm Limited
```

Comment:
I can't comment above, `s_veneers.o` is needed? I cant find it used anywhere.

Reply:
It is needed. It contains the definitions of the tfm veneer functions, which are used to call the tfm services from the non-secure code.